code/modules/surgery/mechanic_steps.dm

```diff
@@ -18,8 +17,6 @@
 /datum/surgery_step/mechanic_open/tool_check(mob/user, obj/item/tool)
 	if(istype(tool))
 		preop_sound = tool.usesound
-	else if(!tool.get_sharpness()) //if its not a tool, then we check if its sharp
```

Comment:
honestly I think if you just made this an `if` instead of `else if` it would make it work as intended (as in, only sharp items work for it)